pkg/types/types.go

```diff
@@ -16,18 +16,25 @@ type ContractBilling struct {
 
 // Counters contains aggregate info about the grid
 type Counters struct {
-	Nodes       int64 `json:"nodes"`
-	Farms       int64 `json:"farms"`
-	Countries   int64 `json:"countries"`
-	TotalCRU    int64 `json:"totalCru"`
-	TotalSRU    int64 `json:"totalSru"`
-	TotalMRU    int64 `json:"totalMru"`
-	TotalHRU    int64 `json:"totalHru"`
-	PublicIPs   int64 `json:"publicIps"`
-	AccessNodes int64 `json:"accessNodes"`
-	Gateways    int64 `json:"gateways"`
-	Twins       int64 `json:"twins"`
-	Contracts   int64 `json:"contracts"`
+	Nodes             int64            `json:"nodes"`
+	Farms             int64            `json:"farms"`
+	Countries         int64            `json:"countries"`
+	TotalCRU          int64            `json:"totalCru"`
+	TotalSRU          int64            `json:"totalSru"`
+	TotalMRU          int64            `json:"totalMru"`
+	TotalHRU          int64            `json:"totalHru"`
+	PublicIPs         int64            `json:"publicIps"`
+	AccessNodes       int64            `json:"accessNodes"`
+	Gateways          int64            `json:"gateways"`
+	Twins             int64            `json:"twins"`
+	Contracts         int64            `json:"contracts"`
+	NodesDistribution map[string]int64 `json:"nodesDistribution" gorm:"-:all"`
+}
+
+// NodesDistribution is the number of nodes per each country
```

Comment:
Should this be in the `db/types.go`? since it's not used by the public api (only the db)